x-pack/legacy/plugins/infra/public/utils/url_state.tsx

```diff
@@ -145,18 +144,21 @@ const encodeRisonUrlState = (state: any) => encode(state);
 export const getQueryStringFromLocation = (location: Location) => location.search.substring(1);
 
 export const getParamFromQueryString = (queryString: string, key: string): string | undefined => {
-  const queryParam = QueryString.decode(queryString)[key];
+  const parsedQueryString: Record<string, any> = url.parseUrlQuery(queryString);
```

Comment:
//nit
Why did you specify the type here (and some other, similar, places)? Isn't it inferred?

Reply:
thanks